### PR TITLE
Finishing up bioinfo kpis

### PR DESCRIPTION
**Pull request by remiolsen**
Will just `self.merge()`. Unless @ewels feel like doing a code review :wink:

### CODE REVIEW

File: make_dashboards/templates/internal/js/dashboard.js [hunk 1]
```diff
@@ -76,11 +76,10 @@ $(function () {
...
             '<span style="color:#4c85bc">'+pl['hiseqX_sequencing_l']+'X</span> lanes in progress';
         make_balance_plot('#rc_finished_balance', pl_l['initial_qc_lanes'],   pl['initial_qc_lanes'],   undefined, pl['initial_qc_lanes']+' lanes in progress');
-        console.log(pl['initial_qc_samples']);
         make_balance_plot('#rc_balance',          pl_l['initial_qc_samples'], pl['initial_qc_samples'], undefined, pl['initial_qc_samples']+' samples in progress');
         make_balance_plot('#lp_balance',          pl_l['library_prep'],       pl['library_prep'],       undefined, pl['library_prep']+' samples in progress');
-        make_balance_plot('#seq_balance',         pl_l['sequencing'],         [pl['miseq_sequencing_l'], pl['hiseq_sequencing_l'], pl['hiseqX_sequencing_l']],         undefined, sequencing_subtext);
         make_balance_plot('#bioinfo_balance',     pl_l['bioinformatics'],     pl['bioinformatics'],     undefined, pl['bioinformatics']+' lanes in progress');
+        make_balance_plot('#seq_balance',         pl_l['sequencing'],         [pl['miseq_sequencing_l'], pl['hiseq_sequencing_l'], pl['hiseqX_sequencing_l']],         undefined, sequencing_subtext);
```
Comment (remiolsen) [inline make_dashboards/templates/internal/js/dashboard.js:82]: @ewels This is the part I spoke about in the meeting. The `pl` object becomes null (or whatever it's called in JS) after this line. This made the `#bioinfo_balance` plot be "Coming soon" even though there was data for it.

Comment (ewels) [reply make_dashboards/templates/internal/js/dashboard.js:82]: Strange.. Why is it making it `null`?

**Comment by ewels** [comment]
Happy for you to self merge if it works 👍 

Please remember to also do a version bump release on GH after merge + deploy.

File: make_dashboards/templates/internal/js/dashboard.js [hunk 2]
```diff
@@ -291,7 +290,6 @@ function make_balance_plot(target, aim, now, prev, subtext){
         }else{
             my_plotlines = Array();
             chroma_colors = chroma.scale(['#AF2323','#259F0B','#4c85cc']).colors(now.length);
-            pl = [];
```
Comment (remiolsen) [inline make_dashboards/templates/internal/js/dashboard.js:294]: @ewels this was the reason.

Comment (ewels) [reply make_dashboards/templates/internal/js/dashboard.js:294]: Cheeky! I wonder what he was doing there...

**Pull request merged**